app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A tool to ease the distribution of very large files"
+DESCRIPTION="Jigsaw Download, or jigdo, is a tool designed to ease the \
+distribution of very large files over the internet such as CD or DVD \
+images. Its aim is to make downloading these images as easy as a click \
+on a direct download link in a browser, while avoiding all the problems \
```

Comment:
Oops, that's the description for [jigdo](https://github.com/haikuports/haikuports/blob/c2a214e6473117adfdbf84d0f792341b30e1bbd7/app-misc/jigdo/jigdo-0.7.3.recipe#L2..L6).

Keeping part of it is probably ok, but you should probably also mention that jigit is the name of the user-friendly wrapper around mkimage.

More info on https://www.einval.com/~steve/software/JTE/:

> **mkimage**, a simple and very fast tool to reconstruct image files from .jigdo and .template files. It doesn't have any logic to cope with downloading missing files, but will list the missing files that are needed. It is also much faster for people (like me!) who already have full local mirrors.
>
> **jigit** is a first attempt at a user-friendly wrapper for mkimage on a user's machine.

(I guess including these whole paragraphs in `DESCRIPTION` would not be ok, but I don't have have a strong opinion here.)

What about:
> jigit aims to be a user-friendly wrapper for mkimage.
> mkimage is a simple and very fast tool to reconstruct image files from .jigdo and .template files. It doesn't have any logic to cope with downloading missing files, but will list the missing files that are needed. It is also convenient in case the user has a partially outdated optical media or ISO image containing at least some files which are not outdated.

Other suggestions for `DESCRIPTION` (and `SUMMARY`) are welcome!.